File: App Reviews/person.java

class Person {

    public class Person(String name, Integer age, double height, double weight) {
        this.name = name;
        this.age = age
        this.height = height;
        this.weight = weight;
    }

    private String getName() {
        return name;
    }

    private Integer getAge() {  
        return age;
    }

    private void setName(String newName) {
        name = newName;
    }

    private void setAge(Integer newAge) {
        age = newAge;
    }

    public static void main(String[] args) {
        Person p = new Person("Bob", 32, 6.0, 180.0);

        System.out.println(p.getName());
        System.out.println(p.getAge());
    }
}

File: person.py
class Person:

    def __init__(self, name, age, height):
        self.name = name
        self.age = age
        self.height = height

    def get_name(self):
        return self.name
    
    def set_name(self, new_name):
        self.name = new_name

    def __str__(self):
        return f'Name: {self.name} Age: {self.age} Height: {self.height}'
    
person = Person('Bob', 32, 6.0)
person2 = Person('Alice', 26, 5.5)
print(person)
print(person2)